selenium_tests/course_info/page_objects/course_info_search_page_object.py

```diff
@@ -0,0 +1,72 @@
+
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.course_info.page_objects.course_info_base_page_object import CourseInfoBasePageObject
+
+
+class CourseSearchPageLocators(object):
+    # List of WebElements found on course search page
+    PAGE_TITLE = (By.CSS_SELECTOR, "h1")
+    COURSE_INFO_SEARCH_BUTTON = (By.CLASS_NAME, "button")
+    COURSE_INSTANCE_TABLE = (By.ID, "courseInstanceDT")
+    SELECT_SCHOOL_DROPDOWN = (By.ID, "dropdownMenuSchools")
+    SELECT_TERM_DROPDOWN = (By.ID, "dropdownMenuTerm")
+    SELECT_YEAR_DROPDOWN = (By.ID, "dropdownMenuYear")
+    SELECT_COURSE_TYPE_DROPDOWN = (By.ID, "dropdownMenuSites")
+    COURSE_INFO_SEARCH_BUTTON_PATH = (By.XPATH, "//button[@type='button']")
+    COURSE_RESULTS_TABLE = (By.ID, "courseInfoDT")
+
+
+class CourseSearchPageObject(CourseInfoBasePageObject):
+
+    def is_loaded(self):
+        """ determine if the page loaded by looking for a specific element on the page """
+        # frame context stickiness is a bit flaky for some reason; make sure we're in the tool_content frame context
+        # before checking for elements on the expected
+        self.focus_on_tool_frame()
+        try:
+            self.find_element(*CourseSearchPageLocators.COURSE_INFO_SEARCH_BUTTON)
```

Comment:
This doesn't seem like a very selective selector -- I would imagine other pages where the driver would find this object, but it might not be the Course Search Page. It appears to be only looking for a button.